src/components/core/DpTableCardList/DpTableCard.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpTableCard from './DpTableCard.vue'
+
+interface IDpTableCard {
+    open: boolean
+}
+
+const meta: Meta<typeof DpTableCard> = {
+    component: DpTableCard,
+    title: 'Components/TableCard',
+    render: (args) => ({
+        components: {
+            DpTableCard
+        },
+        setup() {
+            return { args }
+        },
+        template: `
+          <div>
+            <ul class="o-list o-list--card u-mb">
+              <div class="o-list__item">
```

Comment:
see bove

Reply:
thanks
Done in https://github.com/demos-europe/demosplan-ui/pull/548/commits/0a8415e5b3023fe483afc217ae0e6ab54cfaf3af

Reply:
this was not the DIV i meant.
I was talking about `<div class="o-list__item">`